locales_dev/en/translation.json

```diff
@@ -31,6 +31,7 @@
   "Behind Latest": "Behind Latest",
   "Behind Latest:": "Behind Latest:",
   "Branch:": "Branch:",
+  "Browse available Tasks; give your input.": "Browse available Tasks; give your input.",
```

Comment:
@dvdherron Changes to this file come automatically as part of the build process, to help see when there are user-facing strings (which require translation) that have changed during development. But then like @duggiemitchell said, before merging we manually copy over (and fix, if needed) the translations to `locales/en/translation.json`, which is the translation file actually used in production.